tests/test_translations.py

```
@@ -0,0 +1,176 @@
+
+
+@pytest.mark.parametrize(
+    "dir",
+    TRANSLATION_DIRS,
+)
+def test_langcode_matches(dir: str):
+    _, code = dir.rsplit(".", 1)
+
+    headers = parse_language_headers(TRANSLATIONS_BASE / dir / "strings.po")
+
+    assert "Language" in headers
```

Ruff (0.12.2)

Use of `assert` detected

(S101)